docs/tasks.yml

```diff
@@ -0,0 +1,106 @@
+version: "3"
+
+vars:
+  # Paths
+  G_DOCS_BUILD_DIR: "{{.G_BUILD_DIR}}/docs/html"
+  G_DOCS_VENV_DIR: "{{.G_BUILD_DIR}}/docs-venv"
+  G_NODE_DEPS_DIR: "{{.G_BUILD_DIR}}/docs-node"
+
+tasks:
+  clean:
+    cmds:
+      - "rm -rf '{{.G_DOCS_BUILD_DIR}}'"
+
+  serve:
+    deps:
+      - "http-server"
+      - "site"
+    cmds:
+      - "npm --prefix '{{.G_NODE_DEPS_DIR}}' exec http-server '{{.G_DOCS_BUILD_DIR}}' -c-1"
+
+  site:
+    vars:
+      CHECKSUM_FILE: "{{.G_BUILD_DIR}}/{{.TASK | replace \":\" \"#\"}}.md5"
+      OUTPUT_DIR: "{{.G_DOCS_BUILD_DIR}}"
+    dir: "{{.TASKFILE_DIR}}"
+    deps:
+      - ":init"
+      - task: ":validate-checksum"
+        vars:
+          CHECKSUM_FILE: "{{.CHECKSUM_FILE}}"
+          DATA_DIR: "{{.OUTPUT_DIR}}"
+      - "docs-venv"
+    cmds:
+      - |-
```

Comment:
Even with the clean build options passed to `sphinx-build`, occasionally some non-text / non-style changes do not get reflected after the rebuilding on my end. Shall we invoke cmd `clean` in this task (or create a task named `clean-site` and let both task `clean` and `site` call it) to do clean builds? Currently rebuilding the site doesn't seem too costly, and getting the changes stably reflected in the build is probably more important.